ci-operator/config/stolostron/hypershift-addon-operator/stolostron-hypershift-addon-operator-backplane-2.5.yaml

```diff
@@ -1,19 +1,19 @@
 base_images:
-  stolostron_builder_go1.20-linux:
+  stolostron_builder_go1.23-linux:
     name: builder
     namespace: stolostron
-    tag: go1.20-linux
+    tag: go1.23-linux
 build_root:
   image_stream_tag:
     name: builder
     namespace: stolostron
-    tag: go1.20-linux
+    tag: go1.21-linux
```

Comment:
Is this `1` intentional? Just double checking.

Reply:
@mikeshng Yes, I wanted to make it the same as 2.7.